internal/cloning/cloning.go

```diff
@@ -154,6 +158,7 @@ func createClone(dss *zfs.Dataset, owner string, snap string, r *internal.Resour
 			Name:              "unless-stopped",
 			MaximumRetryCount: 0,
 		},
+		ShmSize: shmSize,
```

Comment:
Why not just use this?
```suggestion
		ShmSize: r.Docker.Shm,
```

Reply:
It won't be set for most resources - but I guess the default can be moved to when we parse the config?

Reply:
Before this PR, `ShmSize` in the `container.HostConfig` were zero, not sure what a sane default should be, but I think that even if we set it to zero now, that would be that same as it were before.

Reply:
Yes good point - have fixed this in a separate PR